ecgtools/builder.py

```diff
@@ -101,7 +101,16 @@ def get_directories(self):
         `ecgtools.Builder`
         """
         pattern = '*/' * (self.depth + 1)
-        dirs = [x for x in self.root_path.glob(pattern) if x.is_dir()]
+
+        if type(self.root_path) is pathlib.PosixPath:
+            dirs = [x for x in self.root_path.glob(pattern) if x.is_dir()]
+
+        elif type(self.root_path) is list:
+            dirs = [x for path in self.root_path for x in path.glob(pattern) if x.is_dir()]
+
+        else:
+            raise ValueError(f'`root_path` must a list or a string. Got {type(self.root_path)}')
```

Comment:
Pydantic's validation should catch during the input validation step... So, I don't think this is needed (I could be wrong though... Double check)

Reply:
Ahh you are right... it catches it upon read in! Sweet!